CMakeLists.txt

```diff
@@ -241,6 +279,37 @@ if(NOT NONET)
   target_link_libraries(devilutionx PRIVATE sodium)
 endif()
 
+if(USE_SDL1)
+  target_link_libraries(devilutionx PRIVATE
+    ${SDL_TTF_LIBRARY}
+    ${SDL_MIXER_LIBRARY}
+    ${SDL_MIXER_LIBRARIES}
+    ${SDL_LIBRARY}
+    ${FREETYPE_LIBRARIES}
+    ${PNG_LIBRARIES})
+  if(WARPOS)
+    target_link_libraries(devilutionx PRIVATE
+      -lmikmod

```

Comment:
Any idea what mikmod is used for? The only formats we support for is wav.

Reply:
the WarpOS SDL_mixer is compiled with mikmod support. This was a lazy attempt to just get it to work on WarpOS. It does work. :) Will remove this as this won't be an issue when compiling all libs within docker. I'm working on a WarpOS docker toolchain.